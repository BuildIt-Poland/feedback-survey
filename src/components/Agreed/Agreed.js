import React from 'react';
import styled from 'styled-components';

import { spacingSmall, spacingMedium } from '../../styles/designTokens';
import media from '../../styles/media';
import Paragraph from '../Layout/Paragraph';

const Wrapper = styled.section`
  margin: ${spacingSmall} auto;

  ${media.tablet`
    margin: ${spacingMedium} auto;
  `};
`;

const Agreed = () => (
  <Wrapper>
    <Paragraph>
      The feedback you provide will be used for individual development and career pathway and forms part of RSR Reward
      and Recognition scheme.
    </Paragraph>
    <Paragraph>
      Feedback will also be used to enhance and evolve individual contribution to your organisation.
    </Paragraph>
    <Paragraph>
      We therefore ask you to provide feedback based on factual evidence only and we may ask you for further information
      in some individual performance feedback.
    </Paragraph>
  </Wrapper>
);

export default Agreed;
