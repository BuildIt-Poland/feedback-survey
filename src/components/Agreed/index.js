import Agreed from './Agreed';

export default Agreed;
