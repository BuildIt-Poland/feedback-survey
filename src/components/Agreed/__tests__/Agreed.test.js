import React from 'react';
import { create } from 'react-test-renderer';

import Agreed from '../Agreed';

describe('COMPONENT - Agreed', () => {
  it('renders correctly', () => {
    const component = create(<Agreed />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
