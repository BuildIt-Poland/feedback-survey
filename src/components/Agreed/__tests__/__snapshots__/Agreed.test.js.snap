// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Agreed renders correctly 1`] = `
<section
  className="sc-bwzfXH iPoewc"
>
  <p
    className="sc-bdVaJa kjwAVT"
  >
    The feedback you provide will be used for individual development and career pathway and forms part of RSR Reward and Recognition scheme.
  </p>
  <p
    className="sc-bdVaJa kjwAVT"
  >
    Feedback will also be used to enhance and evolve individual contribution to your organisation.
  </p>
  <p
    className="sc-bdVaJa kjwAVT"
  >
    We therefore ask you to provide feedback based on factual evidence only and we may ask you for further information in some individual performance feedback.
  </p>
</section>
`;
